Tests/RoslynPluginGeneratorTests/AnalyzerPluginGeneratorTests.cs

```diff
@@ -307,6 +308,98 @@ public void Generate_InvalidSqaleFileSpecified_GeneratorError()
             logger.AssertSingleErrorExists("invalidSqale.xml"); // expecting an error containing the invalid sqale file name
         }
 
+        [TestMethod]
+        public void CreatePluginManifest_TitleMissing()
+        {
+            // When no title is available, ID should be used as a fallback, removing the dot separators for legibility.
+
+            // Arrange
+            DataServicePackage testPackage = CreateTestDataServicePackage();
+            testPackage.Title = null;
+            testPackage.Id = "Foo.Bar.Test";
+
+            // Act
+            PluginManifest actualPluginManifest = AnalyzerPluginGenerator.CreatePluginManifest(testPackage);
+
+            // Assert
+            Assert.IsNotNull(actualPluginManifest);
+            Assert.IsTrue(string.Equals("Foo Bar Test", actualPluginManifest.Name));
```

Comment:
Look at the available overloads for Assert.AreEqual: https://msdn.microsoft.com/en-us/library/microsoft.visualstudio.testtools.unittesting.assert.areequal.aspx